skills/dpdata-cli/SKILL.md

```
@@ -0,0 +1,178 @@
+
+## Quick Start
+
+Run dpdata via uvx:
+
+```bash
+uvx dpdata <from_file> [options]
+```
+
+## Command Line Usage
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)